mlflow/utils/process.py

```diff
@@ -18,8 +18,8 @@ def exec_cmd(
     throw_on_error -- if true, raises an Exception if the exit code of the program is nonzero
     env -- additional environment variables to be defined when running the child process
     cwd -- working directory for child process
-    stream_output -- if true, does not capture standard output and error; if false, captures these
-      streams and returns them
+    capture_output -- if True, captures standard output and error; if False, does not capture
+                      these.
```

Comment:
```suggestion
    capture_output -- if True, captures standard output and error and returns them; if False, does not capture these streams.
```

I think we should indicate standard output and error will be returned.